pippo-session-hazelcast/src/main/java/ro/pippo/session/hazelcast/HazelcastSingleton.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 Herman Barrantes.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ro.pippo.session.hazelcast;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import ro.pippo.core.PippoRuntimeException;
+
+/**
+ * Hazelcast Singleton Resource.
+ *
+ * @author Herman Barrantes
+ * @since 16/05/2016
+ */
+public class HazelcastSingleton {
```

Comment:
Maybe we don't need HazelcastSingleton. I prefer to create the HazelcastInstance in the constructor of MyApplication or onInit. 
I would have done the same notice for above EhcacheFactory class but EhcacheFactory contains a default CacheManager instantiation with some verbose code lines so from this reason I say that it's not a big deal if we keep that class.